microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -354,7 +354,7 @@ See also:
 <!-- ------------------------------ -->
 #### Xbox, HoloLens, and XAML limitations
 
-This version of WebView 2 will only work on PC-class devices, offering the full range of capability found in the WinUI 3 variant. Xbox, HoloLens, and XAML Island support requires additional work, and these devices and scenarios may be considered for future releases.
+This version of WebView 2 will only work on PC-class devices, offering the full range of capability found in the WinUI 3 variant.<!--todo: reword--> Xbox, HoloLens, and XAML Island support requires additional work, and these devices and scenarios may be considered for future releases.
```

Comment:
@nishitha-burman pls click [+/-] "Add a suggestion" icon to enter a rewrite of this line, here and next file below

Reply:
Commented in the above thread ^

Reply:
fixed locally
```
#### XAML limitation

XAML Island support requires additional work and may be considered for future releases.
```